src/helpers/load-package-json.ts

```diff
@@ -0,0 +1,22 @@
+import fs from "fs";
+import path from "path";
+
+export function loadPackageJson(
+  filepath = path.join(process.cwd(), "package.json"),
+): { [key: string]: any } {
+  let pkgContent;
+  try {
+    pkgContent = fs.readFileSync(filepath, "utf8");
```

Comment:
I don't like that we have sync FS access, but that's the case today as well, we can address that in a future change, it's internal anyway.

Reply:
As long as we use express we will have issues because express can not have async handlers.

Reply:
got it, good to know.